src/kernel/arch/x86/64bit/arch.zig

```diff
@@ -332,15 +338,23 @@ pub fn initMem(boot_payload: BootPayload) Allocator.Error!MemProfile {
 }
 
 ///
-/// Initialise the architecture
+/// Initialise the architecture.
 ///
 /// Arguments:
 ///     IN mem_profile: *const MemProfile - The memory profile of the computer. Used to set up
 ///                                         paging.
 ///
 pub fn init(mem_profile: *const MemProfile) void {
-    gdt.init();
-    idt.init();
+    gdt_common.init(gdt_64.Tss, &gdt_64.tss_entry);
+    idt_common.init(idt_64.IdtEntry, &idt_64.table);
+
+    _ = interrupts;
```

Comment:
This line is needed to reference the `getInterruptStub()` function, unless there is a better way of doing it.

Reply:
I suggest adding a comment explaining that so we don't forget why it's there